feat(alerts-service): let `useAlert` return a hide function (LIBS-114) (#820)
dhis2/app-runtime: docs/hooks/useAlert.md

# `useAlert`

`useAlert(message, options?) → { show, hide }`

## Hook arguments

| Name      | Type                   | Description                                                                                                 |
| --------- | ---------------------- | ----------------------------------------------------------------------------------------------------------- |
| `message` | `string` or `Function` | The message to display                                                                                      |
| `options` | `object` or `Function` | A configuration object for the alert. See [note](#note) below for usage instructions in DHIS2 platform-apps |

> ## Usage in DHIS2 platform-apps

Reviewer: ```suggestion
## Usage in apps built on the DHIS2 app platform
```

> The DHIS2 app-shell comes with the alerts provider and a component to show `AlertBar`s in an `AlertStack` (`@dhis2/ui` components), so in a typical DHIS2 platform-app only the `useAlert` hook is used and it can be imported from `@dhis2/app-runtime`.

Reviewer: ```suggestion
The DHIS2 app-shell comes with the alerts provider and a component to show `AlertBar`s in an `AlertStack` (a `@dhis2/ui` components), so in a typical DHIS2 platform app only the `useAlert` hook is used and it can be imported from `@dhis2/app-runtime`.
```

> When used in a platform-app, the `options` argument should be an object with properties that match the [props of of an `AlertBar`](https://ui.dhis2.nu/#/api?id=alertbar).

Reviewer: ```suggestion
When used in an app build on the DHIS2 platform (using `@dhis2/cli-app-scripts`), the `options` argument should be an object with properties that match the [props of an `AlertBar`](https://ui.dhis2.nu/#/api?id=alertbar).
```

> When used in a platform-app, the `options` argument should be an object with properties that match the [props of of an `AlertBar`](https://ui.dhis2.nu/#/api?id=coresrcalertbaralertbarproptypes-object).

Reviewer 2: ```suggestion
When used in a platform-app, the `options` argument should be an object with properties that match the [props of of an `AlertBar`](https://ui.dhis2.nu/#/api?id=alertbar).
```
